lib/plugins/aws/index.js

```diff
@@ -172,14 +172,18 @@ class SDK {
   getServerlessDeploymentBucketName(stage, region) {
     const stackName = `${this.serverless.service.service}-${stage}`;
     return this.request('CloudFormation',
-      'describeStackResource',
+      'describeStacks',
```

Comment:
We've faced a bug in the past when using `describeStacks` vs. `describeStackResource` where our build failed most of the time. Would be great if we can keep `describeStackResource` somehow.

I think that's also the reason why we had the S3 `Output` left in the code but no code which checks for that (`describeStackResource` needs the 'LogicalResourceId` and not the `Output`)